### PR TITLE
fix br bugs in cluster mode

### PR DESCRIPTION
**Pull request by pengweisong**
#### What type of PR is this?
- [x] bug
- [ ] feature
- [ ] enhancement

#### What does this PR do?
1. fix bugs when restore.
2. add an agent error
3. support cmd "show hosts agent"

#### Checklist:
- [ ] Documentation affected (Please add the label if documentation needs to be modified.)
- [ ] Incompatibility (If it breaks the compatibility, please describe it and add the corresponding label.）
- [ ] If it's needed to cherry-pick (If cherry-pick to some branches is required, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

#### Release notes:

Please confirm whether to be reflected in release notes and how to describe:
>                                                                 `

### CODE REVIEW

File: resources/gflags.json [hunk 1]
```diff
@@ -22,4 +23,4 @@
         "rocksdb_column_family_options",
         "rocksdb_block_based_table_options"
     ]
-}
+}
```
Comment (SuperYoko) [inline resources/gflags.json:26]: eof

**Review by Nivras — APPROVED** [review]

**Review by critical27 — APPROVED** [review]
LGTM

File: src/meta/processors/parts/ListHostsProcessor.cpp [hunk 2]
```diff
@@ -133,10 +136,13 @@ nebula::cpp2::ErrorCode ListHostsProcessor::allHostsWithStatus(cpp2::HostRole ro
     }
 
     if (now - info.lastHBTimeInMilliSec_ < FLAGS_removed_threshold_sec * 1000) {
+      int64_t expiredTime = FLAGS_heartbeat_interval_secs * FLAGS_expired_time_factor * 1000;
+      if (info.role_ == cpp2::HostRole::AGENT) {
```
Comment (SuperYoko) [inline src/meta/processors/parts/ListHostsProcessor.cpp:140]: Maybe use if else to control  FLAGS_heartbeat_interval_secs/FLAGS_agent_heartbeat_interval_secs  and calculate once is more clear?

**Review by SuperYoko — DISMISSED** [review]

File: src/meta/processors/parts/ListHostsProcessor.cpp (hunk 2 above)
Comment (pengweisong) [reply src/meta/processors/parts/ListHostsProcessor.cpp:140]: I'll add some comments

**Review by critical27 — DISMISSED** [review]

**Review by SuperYoko — APPROVED** [review]

**Review by critical27 — APPROVED** [review]

**Review by SuperYoko — APPROVED** [review]

**Pull request merged**